utils/mock-data-loader/portal/users.js

```diff
@@ -24,6 +25,26 @@ const USERS = [
     roles: ['maker'],
     bank: UKEF_TEST_BANK_1,
   },
+  {
+    username: 'BANK3_GEF_MAKER1',
+    password: 'AbC!2345',
+    firstname: 'Benito',
+    surname: 'Sutton',
+    email: 'maker2@ukexportfinance.gov.uk',
+    timezone: 'Europe/London',
+    roles: ['maker'],
+    bank: UKEF_GEF_ONLY_BANK,
+  },
+  {
+    username: 'BANK3_GEF_CHECKER1',
+    password: 'AbC!2345',
+    firstname: 'Tony',
+    surname: 'Sheridan',
+    email: 'maker2@ukexportfinance.gov.uk',
```

Comment:
For both these users need to check what email Caroline would like to use

Reply:
Caroline is fine with this email address